src/_guides/language/effective-dart/style.md

```diff
@@ -273,6 +273,29 @@ ID iD;
 {% endprettify %}
 
 
+### PREFER using `_`, `__`, etc. for unused callback parameters.
+
+Sometimes a callback you give to an API is passed an argument that you don't use
+in the body of the function. In that case, it is idiomatic to name the unused
+parameter `_`. If there are multiple unused parameters, use additional
+underscores to avoid name collisions: `__`, `___`, etc.
+
+{:.good}
+<?code-excerpt "misc/lib/effective_dart/style_good.dart (unused-callback-params)"?>
+{% prettify dart tag=pre+code %}
+futureOfVoid.then((_) {
+  print("Operation complete.");
+});
+{% endprettify %}
+
+This guideline is only for *anonymous and local functions* since those are
+usually immediately passed to some other API whose expected function type makes
+it clear what the unused parameter represents. Top-level functions and method
+declarations don't have that context, so should name their parameters. This way,
+someone reading the code knows what that parameter is for, even if not actually
+used.
```

Comment:
```suggestion
This guideline is only for functions that are both *anonymous and local*.
These functions are usually used immediately in a context where it's
clear what the unused parameter represents.
In contrast, top-level functions and method declarations don't have that context,
so their parameters must be named so that it's clear what each parameter is for,
even if it isn't used.
```